test/bootstrap.php

```diff
@@ -26,4 +26,4 @@
 // Use TAO dependency resolver first.
 require_once $extensionRoot . './../vendor/autoload.php';
 
-common_Config::load($extensionRoot . 'config/sample/generis.conf.php');
+common_Config::load($extensionRoot . 'test/generis.conf.php');
```

Comment:
```suggestion
common_Config::load($extensionRoot . 'test/config/generis.conf.php');
```
Wdyt?

Reply:
Good point, makes sense!

I'll update it shortly.